shaders/ltc.hlsl

```diff
@@ -24,8 +24,8 @@ StructuredBuffer<InstanceConstants> instanceConstants;
 StructuredBuffer<RectLight> lights;
 StructuredBuffer<Material> materials;
 
-Texture2D textures[64]; // bindless
-SamplerState samplers[15];
+Texture2D textures[8]; // bindless
```

Comment:
move constants to bindless.h and set for win-mac different values. pass compiler directive in shader manager.